src/ImageAPI/Utils/ImageProcessor.cs

```diff
@@ -26,6 +30,21 @@ public virtual async Task<string> GetBase64ImageAsync()
         return Convert.ToBase64String(imageBytes);
     }
 
+    public virtual void DrawText(Erc721Attribute attribute, TextOptions textOptions, IBrush? brush = null, IPen? pen = null)
+    {
+        string text;
+        if (attribute.DisplayType == DisplayType.String)
+        {
+            text = attribute.Value.ToString()!;
+        }
+        else
+        {
+            text = attribute.DisplayType == DisplayType.Date ? TimeUtils.FromUnixTimestamp((long)attribute.Value).ToString(CultureInfo.InvariantCulture) : attribute.Value.ToString()!;
+        }
+
+        DrawText(text, textOptions, brush, pen);
+    }
```

Comment:
let override the ToString() of the attribute, and here just use it

Reply:
1. `ToString()` return `string`. `DrawText` return nothing.
2. If you mean, "let's migrate L35-L43 lines in method `ToString()`". `ImageProcessor` represents operation with image. Not logical if `ToString()` method of `ImageProcessor` return value of attribute.
3. `ToString()` don't receive any parameters, but i need `Erc721Attribute attribute` for this logic.